app/src/main/res/layout/activity_welcomepage.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This page has...lots of problems:

![image](https://user-images.githubusercontent.com/44113229/170635942-29dfd2f7-dabb-4588-8c46-a272e67d9e0d.png)